app/experimenter/kinto/tasks.py

```diff
@@ -308,6 +308,7 @@ def nimbus_update_paused_experiments_in_kinto():
     metrics.incr("nimbus_update_paused_experiments_in_kinto.completed")
 
 
+@app.task
```

Comment:
Oh, and this was a weird thing that seems maybe a tiny gotcha in tests:

* The code using this function calls a mock
* The code testing this function calls it directly
* But nothing seems to ensure it's actually been registered as a Celery task

So, when I got down to trying to use this code, it took me a tiny bit to figure this out. Not sure we could wrap this sort of thing in tests, but it's a potential crack down in which things can fall.

Reply:
Yeah totally, in general I've found that kind of thing is best left to integration tests, which we'll have forthcoming when we get more support from qa engineering, otherwise you're really just testing configuration and not logic, which doesn't have a ton of benefit in my experience.